README.md

```diff
@@ -31,30 +32,33 @@ stringData:
 EOF
 ```
 
-## Create a config map that defines the resource mappings from oVirt to KubeVirt:
+## Create oVirt resource mappings that defines how oVirt resources are mapped to kubevirt:
 ```bash
 cat <<EOF | kubectl create -f -
-apiVersion: v1
-kind: ConfigMap
+apiVersion: v2v.kubevirt.io/v1alpha1
+kind: OvirtResourceMapping
```

Comment:
What do you think about source infrastructure agnostic name? We could reuse this mapping CR for other imports.

Reply:
I'm not sure about that. I'd prefer it be provider's specific since other providers might identify their resources differently and even in ovirt not all resources are identified the same way.

Reply:
based on planning to support other provider types and in order to comply with multi-provider support in other resources (vmimport, secret data) the resource is changed to ResourceMapping and added ResourceMapping.spec.ovirt for ovirt's mappings.